bpf/process/types/basic.h

```diff
@@ -1520,6 +1520,7 @@ FUNC_INLINE size_t type_to_min_size(int type, int argm)
 struct match_binaries_sel_opts {
 	__u32 op;
 	__u32 map_id;
+	__u32 mbset_id;
```

Comment:
not a big fan of the naming here given it's already in the match binary selector option and it does not stand out that it's for the follow children feature. In general I think `mbset` for match binaries set does not give a strong idea it's behind the "follow children" feature.

Reply:
I have a different opinion. The mechanism is a matchBinaries set (`mbset`) and is used to implement the `followChidren` feature. I prefer to name things based on what they are rather than how they are used. Note that the latter can change. In any case, I'm happy to change it. What would be your suggestion of a better name?